helm/pay-wallet-values-dev.yaml

```diff
@@ -81,6 +81,11 @@ microservice-chart:
     SCHEDULER_CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS: "0"
     SCHEDULER_CDC_SEND_RETRY_MAX_ATTEMPTS: "1"
     SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS: "1000"
+    CDC_REDIS_RESUME_KEYSPACE: "pay-wallet-scheduler"
+    CDC_REDIS_RESUME_TARGET: "pay-wallet-job"
+    CDC_REDIS_RESUME_FALLBACK_IN_MIN: "10"
+    CDC_REDIS_RESUME_TTL_IN_MIN: "120"
+    CDC_RESUME_SAVE_INTERVAL: "5"
```

Comment:
Please remove save interval, the job will be responsible to handle the configuration for saving checkpoints.
Please consider using the prefix SCHEDULER_ for these variables and removing CDC_REDIS_RESUME_TARGET, the job id will be passed to service's method as an input parameter

Reply:
done it here 13e819008aa9a134136c7ae5ba42caba884e9c93